### PR TITLE
Hide forms and show them when the user is logged in

### PR DESCRIPTION
**Pull request by shaargtz**
# What does this PR do?
- Hides HTML forms from people who are not logged in

**Related Issues**
- #66 

---
**Merge Checklist**
- [x] Branch is up to date
- [ ] No code style issues

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -91,11 +91,28 @@ function fetchAuthentication() {
...
+
+      const commentSubmission = document.getElementsByClassName('post-comment');
+      if (commentSubmission != null) {
+        // The timeout is to wait for the dynamically generated forms of each
+        // answer to appear in the DOM.
+        setTimeout(() => {
+          for (element of commentSubmission) {
+            element.style.display = "block";
+          }
+        }, 500);
```
Comment (abundis28) [inline src/main/webapp/script.js:113]: Could you explain a little more about the timeout (for example the value 500 included at the end).

Comment (shaargtz) [reply src/main/webapp/script.js]: done

File: src/main/webapp/question.html [hunk 2]
```diff
@@ -68,7 +68,7 @@
       <!-- Question data -->
       <div id="question"></div>
       <!-- Answer submission -->
-      <div id="post-answer">
+      <div id="post-answer" style="display: none;">
```
Comment (Domene99) [inline src/main/webapp/question.html:71]: nit: don't inline style

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -91,11 +91,30 @@ function fetchAuthentication() {
...
 
-      // Show question submission box when logged in.
+      // Show submission forms when logged in.
       const questionSubmission = document.getElementById('post-question');
       if (questionSubmission) {
         questionSubmission.style.display = "block";
       }
+
+      const answerSubmission = document.getElementById('post-answer');
+      if (answerSubmission) {
```
Comment (Domene99) [inline src/main/webapp/script.js:101]: Non consistent if formatting with L80

File: src/main/webapp/question.html (hunk 2 above)
Comment (shaargtz) [reply src/main/webapp/question.html]: done!

File: src/main/webapp/script.js (hunk 3 above)
Comment (shaargtz) [reply src/main/webapp/script.js:104]: done!

File: src/main/webapp/script.js [hunk 4]
```diff
@@ -97,11 +97,30 @@ function fetchAuthIndexQuestion() {
...
+
+      const commentSubmission = document.getElementsByClassName('post-comment');
+      if (commentSubmission != null) {
+        // The timeout is to wait for the dynamically generated forms of each
+        // answer to appear in the DOM so that the attribute can be changed.
+        setTimeout(() => {
+          for (element of commentSubmission) {
+            element.style.display = "block";
+          }
+          // The timeout of 500ms is enough to let the forms load and not make
```
Comment (timothy-parsons) [inline src/main/webapp/script.js:119]: Fine for now, but just FYI in practice you might see this fail for users with poor connections.

Comment (shaargtz) [reply src/main/webapp/script.js:119]: yeah, I know it isn't the best practice to use timeouts and I should do some async functions but I didn't think of a flow that was appropiate

**Pull request merged**